### ISSUE TITLE
Tutorial request for hybrid model (word+character) 

### ISSUE THREAD

**Issue by Raghava14** [tensorflow/tensorflow#14018]
The implementation done in the paper:
http://aclweb.org/anthology/P/P16/P16-1100.pdf

is a hybrid seq2seq model with advancements where the encoder is fed with inputs based on following two cases:

1.Normal vector representation of a word (Embedding vector) - when the word input is present in the vocabulary

2.Output of another LSTM network - when the word is **out of vocabulary** and a separate character based LSTM is used to **generate an embedding on the fly**

Consider the following example sentence:
"The brown fox jumped over the lazy dog"

Assume these are the words present in the vocabulary: _The, brown, jumped, over, dog_ - These words are fed to the seq2seq encoder as such

out of vocabulary(OOV) words are: _fox, lazy_ - These words are passed to a character LSTM and the output of the same is passed to the seq2seq model along with the above words

These both word level and character level encoder needs to be trained end to end simultaneously. 

Since the implementation is a bit different from the normal seq2seq can a tutorial or example of such case be added to the examples section?

**Comment by skye** [comment]
@wolffg FYI

**Comment by Jake8K** [comment]
second!
I'm trying to implement this model in tf, ( https://arxiv.org/abs/1606.01700 ) which receives the same word at the word and character level to a lookup table and bidirectional LSTM (respectively), the output of both is then concatenated using a "gate" function before sending the imbedding to another LSTM.
Their Theano source-code is here: https://github.com/ nyu-dl/gated_word_char_rlm.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by skye** [comment]
@nealwu maybe a good suggestion for the model garden?

**Comment by nealwu** [comment]
Thanks, we'll keep this in mind.

**Comment by tensorflowbutler** [comment]
A member of the TensorFlow organization has replied after the stat:awaiting tensorflower label was applied.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 112 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 127 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 142 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 157 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 172 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 187 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by badlogicmanpreet** [comment]
Hi @MarkDaoust  is this issue being worked on or can I give it a try? Thanks

**Comment by tensorflowbutler** [comment]
It has been 29 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
